tests/Tests.AzureAppConfiguration/Tests.AzureAppConfiguration.csproj

```diff
@@ -11,21 +11,35 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Identity" Version="1.10.2" />
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="5.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
-    <PackageReference Include="Moq" Version="4.16.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
     <PackageReference Include="System.Linq.Async" Version="6.0.1" />
-    <PackageReference Include="xunit" Version="2.4.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+	<PackageReference Include="Moq" Version="4.20.70" />
+	<PackageReference Include="xunit" Version="2.7.0" />
+	<PackageReference Include="xunit.runner.visualstudio" Version="2.5.7">
+	  <PrivateAssets>all</PrivateAssets>
+	  <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+	</PackageReference>
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\Microsoft.Extensions.Configuration.AzureAppConfiguration\Microsoft.Extensions.Configuration.AzureAppConfiguration.csproj" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'net48'">
+	<PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="3.1.32" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+	<PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
   </ItemGroup>
```

Comment:
We did not have this distinction of net48 vs net6 earlier. Can these frameworks still reference the same package version?

Reply:
They can, I could just update the version to 8.0.0 where it is currently. I only did this because I was looking at the [target .NET 8 PR](https://github.com/microsoft/FeatureManagement-Dotnet/pull/365) in the feature management repo, but it's not necessary.

Reply:
what is the latest version that works for all frameworks? If the split isn't necessary, I'd vote for keeping it simple and use the same version everywhere. Its only the test project anyway.

Reply:
8.0.0 works when I run locally